UnityEngine.AssetBundles/Editor/AssetBundleBrowser/InspectTab/AssetBundleInspectTab.cs

```diff
@@ -17,28 +17,67 @@ public class AssetBundleInspectTab
         private InspectTabData m_Data;
         
 
-        private List<string> m_BundleList = new List<string>();
+        private List<string> m_BundleList;
         private InspectBundleTree m_BundleTreeView;
         [SerializeField]
         private TreeViewState m_BundleTreeState;
 
         public Editor m_Editor = null;
 
-        //[SerializeField] 
-        private List<AssetBundle> m_LoadedBundles;
-
         private SingleBundleInspector m_SingleInspector;
 
+        /// <summary>
+        /// Collection of loaded asset bundles indexed by bundle name
+        /// </summary>
+        private Dictionary<string, AssetBundle> m_loadedAssetBundles;
+
+        /// <summary>
+        /// Returns a loaded asset bundle by name if the bundle exists in our container.
+        /// </summary>
+        /// <returns>Asset bundle instance if loaded, otherwise null.</returns>
+        /// <param name="bundleName">Name of the loaded asset bundle.</param>
+        private AssetBundle GetLoadedBundleByName(string bundleName)
+        {
+            if (string.IsNullOrEmpty(bundleName))
+            {
+                return null;
+            }
+
+            if (!m_loadedAssetBundles.ContainsKey(bundleName))
+            {
+                return null;
+            }
 
-        public AssetBundleInspectTab()
+            return m_loadedAssetBundles[bundleName];
+        }
+
+        /// <summary>
+        /// Removes the file extension from the path if it exists.
+        /// </summary>
+        /// <returns>The path without file extension.</returns>
+        /// <param name="path">The input path.</param>
+        public static string GetPathWithoutFileExtension(string path)
```

Comment:
You're right. I forgot to remove this when I stopped using it.
Will be removed in the next commit.